Easydict/NewApp/View/SettingView/Tabs/GeneralTab.swift

```diff
@@ -197,6 +225,19 @@ struct GeneralTab: View {
 
     @Default(.fontSizeOptionIndex) private var fontSizeOptionIndex
     @Default(.selectedMenuBarIcon) private var selectedMenuBarIcon
+
+    private var detectSetShortcutStatus: Bool {
```

Comment:
IMHO, this could be rename to `shortcutsHaveSetuped` like below:

```
private var shortcutsHaveSetuped: Bool {
    Defaults[.inputShortcut] != nil || Defaults[.selectionShortcut] != nil
}
```

It will look more clear. What do you think?

Reply:
okay you are right